aikau/src/main/resources/alfresco/renderers/css/PublishAction.css

```diff
@@ -1,8 +1,29 @@
 .alfresco-renderers-PublishAction {
 
    cursor: pointer;
-   
+
    &__image {
       vertical-align: text-bottom;
    }
+}
+.alfresco-lists-views-layout-_MultiItemRendererMixin__item {
+   &:hover {
+      .alfresco-renderers-PublishAction.hover-only {
+         > span {
+            visibility: visible;
+         }
+         > img {
```

Comment:
In addition to using BEM (as described above), these two selectors can be combined, i.e. `> span, > img {`